emails/utils.py

```diff
@@ -61,20 +64,109 @@ def ses_send_email(from_address, to_address, subject, message_body):
     return HttpResponse("Sent email to final recipient.", status=200)
 
 
-def ses_relay_email(from_address, relay_address, subject, message_body):
+@time_if_enabled('ses_send_email')
+def ses_send_raw_email(
+        from_address, to_address, subject, message_body, attachments):
```

Comment:
This will get complained on flake8 because the parameters have the same visual indentation as the closing parenthesis. I will apply:
```
def ses_send_raw_email(
                from_address, to_address, subject, message_body, attachments
        ):
```
instead.

Reply:
Example from [Black](https://pypi.org/project/black/) seem to use different styling as seen in their [example](https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ARIAmpdAD2IimZxl1N_WlkPinBFoXIfdFTaTVkGVeHShArYj9yPlDvwBA7LhGo8BvRQqDilPtgsfdKl-ha7EFp0Ma6lY_06IceKiVsJ3BpoICJM9wU1VJLD7l3qd5xTmo78LqThf9uibGWcWCD16LBOn0JK8rhhx_Gf2ClySDJtvm7zQJ1Z-Ipmv9D7I_zhjztfi2UTVsJp7917XToHBm2EoNZqyE8homtGskFIiif5EZthHQvvOj8S2gJx8_t_UpWp1ScpIsD_Xq83LX-B956I_EBIeNoGwZZPFC5zAIoMeiaC1jU-sdOHVucLJM_x-jkzMvK8Utdfvp9MMvKyTfb_BZoe0-FAc2ZVlXEpwYgJVAGdCXv3lQT4bpTXyBwDrDVrUeJEg5cXH4TUTNf-yo029ofjTcZgdwbwkBGElHbHHsQNOhuA4R9GbE2Xx6TfVmH9I4AsqU3ohV7t3GkBwkM8XInLiVOQZ4p5yjM-SW4u3I6_BUS8o2djSZaPvzZPDScXVk1OXu3w0wV7DfrgiK_dpzHntoOvqSHrNLg-Ea6zvV6G2nil3QBTBPl5PDtMwDKchtvwmlhnbvTOrh53X9EnSe8QtRKbMLO4pxx4bAJX-hVCXl5OHpCGZLowD7JdKj1-NctJy9DL99yr-X6yu7KwGCYG7t3fm-lt7Lg_HS9xbBrWDVKBKwM2F7hmR1_n9RFjznRBHD3OpHKlgiWjbWJI0Q6GhXazSt_NVH1KtFiY_UPzzuchkeq2AcjGvQd3-ZPkoFJkNU1Xx7q1i62bM0OKwmDiCfvlesDkc1kV7ClOmjkkdP1i8SGDUCylH227AAAAAD6PNxa0rsHuAAGGBckIAABcXArGscRn-wIAAAAABFla) for `very_important_function()` declaration. I will use:
```
def ses_send_raw_email(
        from_address, to_address, subject, message_body, attachments
):
```
instead.